karapace/kafka_rest_apis/consumer_manager.py

```diff
@@ -495,47 +505,44 @@ async def fetch(self, internal_name: Tuple[str, str], content_type: str, formats
                         content_type=content_type,
                     )
                 LOG.debug("Successfully polled for messages")
-                for topic, records in data.items():
-                    for rec in records:
-                        message_count += 1
-                        read_bytes += max(0, 0 if rec.key is None else len(rec.key)) + max(
-                            0, 0 if rec.value is None else len(rec.value)
-                        )
-                        poll_data[topic].append(rec)
-                        read_buffered = True
+                message_count += 1
+                read_bytes += max(0, 0 if message.key() is None else len(message.key())) + max(
+                    0, 0 if message.value() is None else len(message.value())
```

Comment:
why max between 0 and the `len()`, could it deliver a negative number? and when if yes?

Reply:
idem for the other max, could a sum between a `len` and another `max` between 0 and another `len` be negative?